uploadNeuroDB/mass_jiv.pl

```diff
@@ -30,17 +32,21 @@
      ["-verbose", "boolean", 1,   \$verbose, "Be verbose."],
      );
 
-GetOptions(\@arg_table, \@ARGV) ||  exit 1;
+GetOptions(\@arg_table, \@ARGV) ||  exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
 
 # checking for profile settings
+if ( !$profile ) {
+    print STDERR "$Usage\n\tERROR: missing -profile argument\n\n";
+    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
+}
 if(-f "$ENV{LORIS_CONFIG}/.loris_mri/$profile") {
 	{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
 }
-if ($profile && !@Settings::db) {
-    print "\n\tERROR: You don't have a configuration file named '$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n"; exit 33;
-} 
+if ( !@Settings::db ) {
+    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
+                 . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;}
```

Comment:
just out of curiosity, what if the file `prod` does not exist?
we only check if the argument is given and then if the database section is in the prod file, but never see if the prod file is there; as is done here on line 42 above (`if (-f ...)')

Reply:
yeah. That is bad... I see it is like that in most scripts. :S Should I change it in this PR or create a redmine ticket to change this in another PR? What do you think is the best path? I could do either :).

Reply:
no it could be done in a separate one; otherwise this PR would be fixing everything wrong in the MRI scripts and you won't finish anytime soon:) just wanted to raise this comment as I saw it when reviewing the changes...